packages/core/src/ruleset/__tests__/__fixtures__/overrides/aliases/scope.ts

```diff
@@ -0,0 +1,58 @@
+import { DiagnosticSeverity } from '@stoplight/types';
+import {falsy, pattern, truthy} from '@stoplight/spectral-functions';
+import { RulesetDefinition } from '@stoplight/spectral-core';
+
+export { ruleset as default };
```

Comment:
wtf I did not know you could call `export` on an identifier before the identifier is declared. Makes sense, but man that's gotta be a pain on the module loader devs.